File: URI/Iniciante/Python/1064 - Positivos e Média.py
numero = []
a = 0
b = 0

for i in range(6):
  numero.append(float(input()))
 
for x in range(1,7):
    if numero[x-1] > 0:
        b += 1
        a += numero[x-1]   
        
media = a / b
print("%i valores positivos"%(b))
print("%.1f"%(media))

File: URI/Iniciante/C/1064 - Positivos e Média.c
#include <stdio.h>
int main()
{
float n1,n2,n3,n4,n5,n6,media,soma;
int x;
x=0;
soma=0;
scanf("%f %f %f %f %f %f",&n1,&n2,&n3,&n4,&n5,&n6);
    if(n1>0)
    {
        soma=soma+n1;
        x++;
    }
    if(n2>0)
    {
        soma=soma+n2;
        x++;
    }
    if(n3>0)
    {
        soma=soma+n3;
        x++;
    }
    if(n4>0)
    {
        soma=soma+n4;
        x++;
    }
    if(n5>0)
    {
        soma=soma+n5;
        x++;
    }
    if(n6>0)
    {
        soma=soma+n6;
        x++;
    }
    media=soma/x;
    printf("%d valores positivos\n%.1f\n",x,media);
    return 0;
}
